### ISSUE TITLE
TypeError: Failed to construct 'Response': Response with null body status cannot have body

### ISSUE THREAD

**Issue by HunterECampbell** [zoubingwu/msw-auto-mock#23]
I have a DELETE endpoint that returns a success status code of 204.  I've been getting a "null body" error and have tried using `try catch` and `if else` to catch the error, but they aren't working.  Removing `ctx.json(null)` for 204 codes fixes the problem.  Here's the error I've been getting with my DELETE 204:

```
createRequestListener.ts:56 Uncaught (in promise) TypeError: Failed to construct 'Response': Response with null body status cannot have body
    at Object.onMockedResponse (createRequestListener.ts:56:38)
    at handleRequest (handleRequest.ts:127:3)
    at async createRequestListener.ts:34:7
```

**Comment by HunterECampbell** [comment]
I created a PR for this issue.  #22

**Comment by jimmywalsh** [comment]
I've been experiencing this same issue, would love to see see this issue resolved.

**Comment by zoubingwu** [comment]
Thanks! Pull request is merged and package released in v0.12.0
